www/content/attributes/hx-swap.md

```diff
@@ -2,12 +2,13 @@
 title = "hx-swap"
 +++
 
-The `hx-swap` attribute allows you to specify how the response will be swapped in relative to the
-[target](@/attributes/hx-target.md) of an AJAX request.
+The `hx-swap` attribute allows you to specify optionally how the response will be swapped in relative to the
```

Comment:
No need to say "specify optionally" here, I think just leaving it as "allows you to specify" is cleaner.

Reply:
Done.